src/main/antora/resources/antora-resources/antora.yml

```
@@ -24,7 +24,7 @@ asciidoc:
     groupIdStarter: org.springframework.boot
     artifactIdStarter: spring-boot-starter-data-neo4j
     docs-neo4j-docker-version: 5
-    spring-boot-version: 3.2.0
+    spring-boot-version: 3.3.0
```

```suggestion
    spring-boot-version: 3.4.5
```

I think it's totally ok, to go with the lastest one, or?